plugins/ufm_consumer_plugin/scripts/init.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Copyright © 2013-2023 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# This software product is a proprietary product of Nvidia Corporation and its affiliates
+# (the "Company") and all right, title, and interest in and to the software
+# product, including all associated intellectual property rights, are and
+# shall remain exclusively with the Company.
+#
+# This software product is governed by the End User License Agreement
+# provided with the software product.
+# @author: Anan Al-Aghbar
+# @date:   Jan 29, 2023
+#
+# ================================================================
+# This script prepares and checks ufm_consumer docker container Environment
+# ================================================================
+
+set -eE
+PLUGIN_NAME=ufm_consumer
+SRC_DIR_PATH=/opt/ufm/ufm_plugin_${PLUGIN_NAME}/${PLUGIN_NAME}_plugin
+CONFIG_PATH=/config
+
+cp $SRC_DIR_PATH/conf/${PLUGIN_NAME}_httpd_proxy.conf $SRC_DIR_PATH/conf/${PLUGIN_NAME}_plugin.conf $SRC_DIR_PATH/conf/${PLUGIN_NAME}_ui_conf.json ${CONFIG_PATH}
+cp -r $SRC_DIR_PATH/${PLUGIN_NAME}_ui /data
+
+touch ${CONFIG_PATH}/${PLUGIN_NAME}_shared_volumes.conf
+
+echo /opt/ufm/files/log/:/log > ${CONFIG_PATH}/${PLUGIN_NAME}_shared_volumes.conf
+# Update gv.cfg
+/config_consumer.sh
+# UFM version test
+required_ufm_version=(6 12 0)
```

Comment:
the required UFM version should be the version that supports the multisubnet

Reply:
fixed